ramls/cancellation-reason.json

```diff
@@ -17,6 +17,10 @@
     "requiresAdditionalInformation": {
       "type": "boolean"
     },
+    "source": {
+      "description": "Origin of the cancellation reason record, i.e. 'System' or 'User'",
```

Comment:
Maybe lets rename to: Origin of the cancellation reason record, i.e. 'System', 'User', 'Consortium' etc.
Just to show that it is not only 2 possible values exists

Reply:
I have updated the description.